sov-modules/sov-app-template/src/lib.rs

```diff
@@ -155,27 +151,21 @@ where
                         batch_workspace = batch_workspace.commit();
                     }
                     Err(e) => {
-                        // TODO: all the previous "successful" txs will be reverted, is that ok?
                         self.revert_and_slash(batch_workspace);
                         panic!("Demo app txs must succeed but failed with err: {}", e)
```

Comment:
Now that we have the option of returning an error, do we still want to panic here?

Reply:
Let's leave it for now, I want to write a specific test for this (and then I will remove the panic), but first we need to fix the other bug (#179)